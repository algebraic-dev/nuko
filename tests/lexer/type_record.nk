type Ata a = { ata: Int }