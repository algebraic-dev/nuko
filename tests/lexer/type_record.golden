(type,1:0-1:4)
(upper(Ata),1:5-1:8)
(lower(a),1:9-1:10)
('=',1:11-1:12)
('{',1:13-1:14)
(lower(ata),1:15-1:18)
(':',1:18-1:19)
(upper(Int),1:20-1:23)
('}',1:24-1:25)
($EOF,1:25-1:25)
